src/Schema/Directives/Fields/DeprecatedDirective.php

```diff
@@ -29,15 +29,14 @@ public function name()
      *
      * @throws \Nuwave\Lighthouse\Exceptions\DirectiveException
      */
-    public function handleField(FieldValue $value, Closure $next)
+    public function handleField(FieldValue $value, Closure $next): FieldValue
     {
         $reason = $this->directiveArgValue('reason');
 
         if (! $reason) {
-            $parentName = $value->getParentName();
-            $message = "The @{$this->name()} directive requires a `reason` argument [defined on {$parentName}]";
-
-            throw new DirectiveException($message);
+            throw new DirectiveException(
```

Comment:
We should prob. define in our `contributing.md` that when we throw exceptions if the message should have a `.` at the end or not.

Reply:
https://github.com/nuwave/lighthouse/issues/655